### PR TITLE
Replace await/result with proper call

### PR DESCRIPTION
**Pull request by SeanFeldman**
@dpaquette @MisterJames please review

### CODE REVIEW

**Comment by SeanFeldman** [comment]
Please update #549 before when this is merged.

File: AllReadyApp/Web-App/AllReady/DataAccess/IAllReadyDataAccess.cs [hunk 1]
```diff
@@ -98,10 +98,7 @@ public interface IAllReadyDataAccess
 
         #region Skill CRUD
         IEnumerable<Skill> Skills { get; }
-        Skill GetSkill(int skillId);
-        Task AddSkill(Skill value);
-        Task DeleteSkill(int id);
-        Task UpdateSkill(Skill value);
+
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady/DataAccess/IAllReadyDataAccess.cs]: Yay!

**Comment by dpaquette** [comment]
This looks good. Ready to merge assuming the build goes through ok

**Comment by SeanFeldman** [comment]
They say if you ask universe, it will happen. Let's test that :)

**Comment by SeanFeldman** [comment]
:dancers:

**Comment by tonysurma** [comment]
I think @dpaquette comment was equivalent to :shipit: so going to merge

**Pull request merged**